float64 probability
int64 xmin
int64 ymin
int64 xmax
int64 ymax
int16 num
string Class
float64 mean_distance
float64 heading_angle

